lib/slashdeploy/service.rb

```diff
@@ -45,6 +45,7 @@ def create_auto_deployment(environment, sha, user)
           auto_deployment.slack_account, \
           AutoDeploymentCreatedMessage, \
           auto_deployment: auto_deployment
+        DeploymentWatchdogWorker.perform_in(30.seconds, auto_deployment.id)
```

Comment:
Can you throw the `30.seconds` part into a constant somewhere?

Also, 30 seconds is way too short for what this PR is actually testing for; checking if an auto deployment is still pending. It's pretty normal for tests in CI to take 10-20 minutes, so we'd need something a little more reasonable (maybe like 1 hour?).

The 30 second value we came up with was for the scenario where a GitHub deployment is created, but nothing creates any deployment status events.